From b2319ee76dea352fef081ae028e5edb899b96994 Mon Sep 17 00:00:00 2001
From: Michal Hrusecky <Michal@Hrusecky.net>
Date: Fri, 16 Feb 2018 11:11:45 +0100
Subject: [PATCH] feeds: Support our repo paths
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

We are using different paths in our repositories, so customizing build to
reflect that.

Signed-off-by: Michal Hrusecky <Michal@Hrusecky.net>
Signed-off-by: Karel Kočí <karel.koci@nic.cz>
---
 include/feeds.mk | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/include/feeds.mk b/include/feeds.mk
index 2b259b96b3..0f69d508bd 100644
--- a/include/feeds.mk
+++ b/include/feeds.mk
@@ -33,14 +33,14 @@ endef
 # 1: destination file
 define FeedSourcesAppend
 ( \
-  echo 'src/gz %d_core %U/targets/%S/packages'; \
+  echo 'src/gz %d_core %U/core'; \
   $(strip $(if $(CONFIG_PER_FEED_REPO), \
-	echo 'src/gz %d_base %U/packages/%A/base'; \
+	echo 'src/gz %d_base %U/base'; \
 	$(if $(filter %SNAPSHOT-y,$(VERSION_NUMBER)-$(CONFIG_BUILDBOT)), \
 		echo 'src/gz %d_kmods %U/targets/%S/kmods/$(LINUX_VERSION)-$(LINUX_RELEASE)-$(LINUX_VERMAGIC)';) \
 	$(foreach feed,$(FEEDS_AVAILABLE), \
 		$(if $(CONFIG_FEED_$(feed)), \
-			echo '$(if $(filter m,$(CONFIG_FEED_$(feed))),# )src/gz %d_$(feed) %U/packages/%A/$(feed)';)))) \
+			echo '$(if $(filter m,$(CONFIG_FEED_$(feed))),# )src/gz %d_$(feed) %U/$(feed)';)))) \
 ) >> $(1)
 endef

--
2.25.1
